spec/integration_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'aliquot'
+require 'aliquot/payment'
+
+require 'aliquot-pay'
+
+shared_examples 'common integration tests' do
+  context 'invokes' do
+    it Aliquot::Validator::Token do
+      e = StandardError.new('stub method')
+      allow(Aliquot::Validator::Token).to receive(:new).and_raise(e)
+
+      is_expected.to raise_error(e)
+
+      expect(Aliquot::Validator::Token).to have_received(:new).with(token)
+    end
+
+    it Aliquot::Validator::SignedMessage do
+      e = StandardError.new('stub method')
+      allow(Aliquot::Validator::SignedMessage).to receive(:new).and_raise(e)
+
+      is_expected.to raise_error(e)
+
+      expect(Aliquot::Validator::SignedMessage).to have_received(:new).with(generator.build_signed_message)
+    end
+
+    it Aliquot::Validator::EncryptedMessageValidator do
+      e = StandardError.new('stub method')
+      allow(Aliquot::Validator::EncryptedMessageValidator).to receive(:new).and_raise(e)
+
+      is_expected.to raise_error(e)
+
+      expect(Aliquot::Validator::EncryptedMessageValidator).to have_received(:new).with(generator.build_cleartext_message)
+    end
+  end
+end
+
+describe Aliquot::Payment do
+  let (:token) { generator.token }
```

Comment:
Is `token` used anywhere? Should it have been used in line 41? (Rather than `generator.token`.)

Reply:
It is used on line 14, but I agree with your change.